manifests/instance.pp

```diff
@@ -163,14 +163,24 @@
     path    => "/etc/redis/${redis_port}.conf",
     mode    => '0644',
     content => template('redis/redis_port.conf.erb'),
-    replace => $manage_config_file,
```

Comment:
why was this forced true?   *confused*  why couple the manage_config_files and the restart on change settings?